web/views.py

```
@@ -4644,77 +4644,113 @@
     return render(request, "teams/delete_confirm.html", {"goal": goal})
 
 
+@login_required
 @teacher_required
 def add_student_to_course(request, slug):
```

Ruff (0.11.9)

Missing return type annotation for public function `add_student_to_course`

(ANN201)